src/FixedDataTableWidthHelper.js

```diff
@@ -14,137 +14,198 @@
 
 import React from 'React';
 
+/**
+ * Flatten nested columns 
+ */
+function getNestedColumns(/*array*/ columns, /*array*/ initial)/*array*/ {
+  return columns.reduce((prev, column) => {
+    if (!column.type.__TableColumn__) {
```

Comment:
Ya could you just add a comment `// TODO (dangoo) improve this check to only return true for column groups` and put it here and where we have a similar check below.  I think that's good enough for now.